controllers/collection.lua

```diff
@@ -22,55 +22,46 @@
 -- You should have received a copy of the GNU Affero General Public License
 -- along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+local db = package.loaded.db
 local util = package.loaded.util
 local validate = package.loaded.validate
+local json_params = package.loaded.app_helpers.json_params
+local yield_error = package.loaded.app_helpers.yield_error
+
 
 local Users = package.loaded.Users
 local Projects = package.loaded.Projects
 local Collections = package.loaded.Collections
+local CollectionMemberships = package.loaded.CollectionMemberships
 
 require 'responses'
 require 'validation'
 
 -- a simple helper for conditionally setting the timestamp fields
 -- TODO: move to a more useful location.
 local current_time_or_nil = function(option)
-    if option == 'true' then
+    if option == true then
```

Comment:
Can't this just be `if option`? Or do we expect option to be anything other than `false`, `null` or `true`?

Reply:
JSON params are (de)serialized correctly. So `{"a": true}` is the actual value true, but the string true is kept as a string. 

I don't mind treating `true` and `"true"` identically, but seems a bit unsafe for the sharing or publishing state to be loose and accept any truthy value as good to publish. With that in mind I wrote this function to match the state of publishing/sharing, but it is a bit more generic so maybe it doesn't need to match.

Reply:
Got it :)